### ISSUE TITLE
Please release 2.0.2 to publish Click compatibility fix

### ISSUE THREAD

**Issue by MaxG87** [TailorDev/Watson#469]
Since Click 8.0 was published, it is no longer possible to install `watson` using `pipx install td-watson`. That would pull version 2.0.1 and subsequently Click 8.0. It gave me quite a headache to sort this out.

By releasing a bugfix release these complications could be avoided for others.

**Comment by ericguirbal** [comment]
Two workarounds:
 1. Replace click 8.* with the lastest compatible version: ```pipx install td-watson && pipx inject td-watson click==7.1.2```
 2.  Install Watson from the GitHub repo where the bug has been fixed one year ago: ```pipx install git+https://github.com/TailorDev/Watson.git```.

**Comment by MaxG87** [comment]
I didn't know about `pipx inject`! Thank you very much!

I ended up with
```shell
python setup.py bdist_wheel
pipx install dist/td_watson-2.0.1-py2.py3-none-any.whl
```

because I had it cloned anyways.

**Comment by jmaupetit** [comment]
Should be fixed with the 2.1.0 fresh release!
